<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StatusForm</class>
 <widget class="QWidget" name="StatusForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>347</width>
    <height>44</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="1,98,1">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="hpHead">
       <property name="text">
        <string>hp:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="mpHead">
       <property name="text">
        <string>mp:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="PointBar" name="hpBar">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="PointBar" name="mpBar">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="2">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="hpNum">
       <property name="text">
        <string>(0/0/0)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="mpNum">
       <property name="text">
        <string>(0/0/0)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PointBar</class>
   <extends>QLabel</extends>
   <header location="global">statusBar/pointbar.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
